tests/e2e/vsphere_volume_expansion.go

```diff
@@ -730,17 +730,17 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
+		ginkgo.By("Checking for error in events related to pvc " + pvclaim.Name)
 		if !featureEnabled {
 			ginkgo.By("File system resize should not succeed since SPS service is down" +
 				" and cns new sync feature is disabled. Expecting an error")
-			if guestCluster {
-				expectedErrMsg = "didn't find a plugin capable of expanding the volume"
-			} else {
-				expectedErrMsg = "failed to expand volume"
-			}
+			expectedErrMsg = "didn't find a plugin capable of expanding the volume"
```

Comment:
this is different based on the flavors?

Reply:
tested for Guest Cluster and Block Vanilla. Error is coming same "didn't find a plugin capable of expanding the volume"

Reply:
I don't think this is an error